Directory.mk

```diff
@@ -22,11 +22,15 @@ include $(APPDIR)/Make.defs
 
 # Sub-directories that have been built or configured.
 
-SUBDIRS       := $(dir $(wildcard *$(DELIM)Makefile))
-CONFIGSUBDIRS := $(filter-out $(dir $(wildcard *$(DELIM)Kconfig)),$(SUBDIRS))
-CLEANSUBDIRS  += $(dir $(wildcard *$(DELIM).depend))
-CLEANSUBDIRS  += $(dir $(wildcard *$(DELIM).kconfig))
+SUBDIRS       := $(dir $(wildcard */Makefile))
```

Comment:
Why `$(DELIM)` is not working here?

Reply:
result CONFIGSUBDIRS and CLEANSUBDIRS be empty set in Windows native build

Reply:
Is `$(DELIM)` obsolete? Do all operating systems (including Windows) recognize `/` delimiter now?

Reply:
> Is `$(DELIM)` obsolete? Do all operating systems (including Windows) recognize `/` delimiter now?

Windows is not a POSIX environment,  some makefile command was not support in Windows, As suggest in nuttx/README we use GnuWin32  to supply this windows version Makefile commad(for example: wildcard), some special cases we must use / rather than \  ,for here, we must use /, otherwise CONFIGSUBDIRS and CLEANSUBDIRS would be empty set in Windows native build, and will give fatal error, but use / we can build succesfully.